gtsfm/utils/geometry_comparisons.py

```diff
@@ -205,3 +205,50 @@ def compute_relative_unit_translation_angle(U_1: Optional[Unit3], U_2: Optional[
     angle_rad = np.arccos(dot_product)
     angle_deg = np.rad2deg(angle_rad)
     return angle_deg
+
+
+def compute_translation_to_direction_angle(
+    i2Ui1: Optional[Unit3],
+    wTi2: Optional[Point3],
```

Comment:
Thanks for the updates, @akshay-krishnan . Should wTi2 be Optional[Pose3] in args?

Reply:
Good catch! updated.